CMakePresets.json

```
@@ -88,140 +121,73 @@
       }
     },
     {
-      "name": "x64-debug",
-      "displayName": "x64 Debug",
-      "description": "Windows x64 Debug",
-      "inherits": "x64",
+      "name": "debug-build",
+      "displayName": "Debug Build",
+      "description": "Debug Build (Hidden)",
       "hidden": true,
       "cacheVariables": {
         "CMAKE_BUILD_TYPE": "Debug"
       }
     },
     {
-      "name": "x64-release",
-      "displayName": "x64 Release",
-      "description": "Windows x64 Release",
-      "inherits": "x64",
+      "name": "release-build",
+      "displayName": "Release Build",
+      "description": "Release Build (Hidden)",
       "hidden": true,
       "cacheVariables": {
         "CMAKE_BUILD_TYPE": "RelWithDebInfo"
       }
     },
     {
-      "name": "x86-debug",
-      "displayName": "x86 Debug",
-      "description": "Windows x86 Debug",
-      "inherits": "x86",
-      "hidden": true,
-      "cacheVariables": {
-        "CMAKE_BUILD_TYPE": "Debug"
-      }
-    },
-    {
-      "name": "x86-release",
-      "displayName": "x86 Release",
-      "description": "Windows x86 Release",
-      "inherits": "x86",
+      "name": "enable-tests",
+      "displayName": "Enable Tests",
+      "description": "Enable Tests (hidden)",
       "hidden": true,
       "cacheVariables": {
-        "CMAKE_BUILD_TYPE": "RelWithDebInfo"
-      }
-    },
-    {
-      "name": "x64-static-debug",
-      "displayName": "Windows x64 Debug",
-      "description": "Windows x64 Debug",
-      "inherits": "x64-static",
-      "hidden": true,
-      "cacheVariables": {
-        "CMAKE_BUILD_TYPE": "Debug"
-      }
-    },
-    {
-      "name": "x64-static-release",
-      "displayName": "Windows x64 Release, static",
-      "description": "Windows x64 Release, static",
-      "inherits": "x64-static",
-      "hidden": true,
-      "cacheVariables": {
-        "CMAKE_BUILD_TYPE": "RelWithDebInfo"
+        "BUILD_TESTING": true,
+        "ENABLE_PROXY_TESTS": false,
+        "BUILD_SAMPLES": true
       }
     },
     {
-      "name": "x86-static-debug",
-      "displayName": "Windows x86 Debug, static",
-      "description": "Windows x86 Debug, static",
-      "inherits": "x86-static",
-      "hidden": true,
-      "cacheVariables": {
-        "CMAKE_BUILD_TYPE": "Debug"
-      }
-    },
-    {
-      "name": "x86-static-release",
-      "displayName": "Windows x86 Release, static",
-      "description": "Windows x86 Release, static",
-      "inherits": "x86-static",
-      "hidden": true,
-      "cacheVariables": {
-        "CMAKE_BUILD_TYPE": "RelWithDebInfo"
-      }
+      "name": "x86-msvc-static-debug",
+      "displayName": "Windows x86, msvc static, Debug ",
+      "description": "Windows x86, msvc static, Debug",
```

If hiding this was intentional:
```suggestion
      "description": "Windows x86, msvc static, Debug (hidden)",
```